groups/html.yml

```diff
@@ -0,0 +1,8 @@
+# This group isn't intended for everything defined in HTML:
+# spec: https://html.spec.whatwg.org/multipage/
+#
+# It is for the markup language, and not for all APIs and infrastructure
+# defined in HTML. For example, structuredClone() should not be in this group.
+# APIs on HTML*Element and CSS features specific to an HTML element generally
+# do make sense to group under HTML, however.
```

Comment:
I'm slightly anxious about this. Do we contemplate a DOM group existing and how does that impact this?

Reply:
I think DOM would be everything pertaining to the `Node` hierarchy except language-specific interfaces like everything `HTMLElement` and `SVGElement`. I don't expect it will be that messy, but always a little :)

Reply:
However, sometimes people say "DOM APIs" for what I would call "APIs defined using Web IDL", a vastly broader group of things. I don't think that should be a group at all, it's just the collection of all BCD keys under api.*.

Reply:
Ah ha, OK that's fine.